Library/Formula/shairport-sync.rb

```diff
@@ -0,0 +1,56 @@
+class ShairportSync < Formula
+  desc "AirTunes emulator. Shairport Sync adds multi-room capability."
+  homepage "https://github.com/mikebrady/shairport-sync"
+  url "https://github.com/mikebrady/shairport-sync/archive/2.4.1.tar.gz"
+  sha256 "d2af0e9ee86f3c24f08ddb43ee73c79c5426b52a11d83a11f6c71ed2023df32d"
+
+  head "https://github.com/mikebrady/shairport-sync.git"
+
+  depends_on "openssl"
+  depends_on "popt"
+  depends_on "libsoxr"
+  depends_on "libao"
+  depends_on "libdaemon"
+  depends_on "libconfig"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  patch :DATA
+
+  def install
+    system "autoreconf", "-i", "-f"
```

Comment:
Let's condense this to `system "autoreconf", "-fvi"` to keep it consistent with most of the other `autoreconf` calls in the core.